src/python/ssh/sshcp.py

```
@@ -279,20 +279,20 @@ def __run_command(self,
                     'Connection refused',  # i=4, connection refused
                 ])
                 if i > 0:
-                    before = sp.before.decode().strip() if sp.before != pexpect.EOF else ""
-                    after = sp.after.decode().strip() if sp.after != pexpect.EOF else ""
+                    before = sp.before.decode(errors='replace').strip() if sp.before != pexpect.EOF else ""
+                    after = sp.after.decode(errors='replace').strip() if sp.after != pexpect.EOF else ""
                     self.logger.warning("Command failed: '{} - {}'".format(before, after))
```

Ruff (0.15.1)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)